standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/utils/TestMetaStoreUtils.java

```diff
@@ -50,7 +51,7 @@ public class TestMetaStoreUtils {
   private final String date;
   private final String timestamp;
 
-  public TestMetaStoreUtils(String zoneId, LocalDateTime timestamp) {
+  public TestMetaStoreUtils(String zoneId, ZonedDateTime timestamp) {
     this.timezone = TimeZone.getTimeZone(zoneId);
     this.timestamp = timestamp.format(FORMATTER);
```

Comment:
How about keeping `LocalDateTime` in constructor parameter, and generate the timeZone related timestamp string here:
```java
this.timestamp = ZonedDateTime.of(timestamp, ZoneId.of(zoneId)).format(FORMATTER);
```

Reply:
From @zabetak's comment, I have reverted to LocalDateTime.